ReadMe.md

```diff
@@ -0,0 +1,37 @@
+# LevelTemplate
+
+Outpost 2 Level template project for use in creating new Outpost 2 missions.
+
+Missions in Outpost 2 are implemented as DLL (Dynamic Link Library) files. These contain executable code. They are typically produced by a C++ compiler, such as Microsoft Visual C++, included as part of [Visual Studio](https://visualstudio.microsoft.com/). The Visual Studio Community Edition is a free download, albeit a bit hefty in size. It is required to build Outpost 2 missions. Be sure to select the C++ toolset when installing.
```

Comment:
Might be worth noting that they must be compiled in x86 (win32) as x64 bit compilations are incompatible with Outpost 2?

Also might be worth noting that while it must be compiled using the MSVC compiler to remain compatible with Outpost 2 (I think anyways). I suspect from your comments elsewhere mingw does not work. I think code::blocks might work, but I've never used it. I know others in the forums have and suspect it might be able to leverage the MSVC compiler but provide an alternative to VS if anyone cannot bear to use it.

Reply:
Yeah... but that seems like a bit too much info, especially for the lead description. Besides, the platform setting detail is already taken care of by the template project settings. The user never needs to be aware of that.

Alternative compilers and Code Blocks might be worth mentioning, but probably more as an appendix.